README.rst

```diff
@@ -113,12 +105,38 @@ On Windows:
   * ``REMOTE_TARANTOOL_CONSOLE_PORT=3302``.
 * Run ``python setup.py test``.
 
+Build docs
+^^^^^^^^^^
+
+To build documentation, first you must install its build requirements:
+
+.. code-block:: bash
+
+    $ pip install -r requirements-doc.txt
+
+Then run
+
+.. code-block:: bash
+
+    $ make docs
```

Comment:
We need to unify the doc:

1. python setup.py install
2. python setup.py test
but
3. make docs

Reply:
Added a separate commit with this change